docs/ContactUs.adoc

```diff
@@ -2,6 +2,11 @@
 :site-section: ContactUs
 :stylesDir: stylesheets
 
-* *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level3/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
+* *Bug reports, Suggestions* : Post in our https://github.com/AY1920S2-CS2103T-W12-1/main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at
+    ** `e0325707 [at]  u.nus.edu`
+    ** `e0318484 [at] u.nus.edu`
+    ** `jeremyloh [at]  u.nus.edu`
+    ** `e0319018 [at]  u.nus.edu`
+    ** `e0318636 [at]  u.nus.edu`
```

Comment:
I think we should add the names of the person for the email at the start. I think its hard to know who has what email.